src/IntegralArrays.jl

```diff
@@ -1,5 +1,74 @@
 module IntegralArrays
 
-# package code goes here
+using FileIO # for loading images
+using QuartzImageIO, ImageMagick, ImageSegmentation, ImageFeatures # for reading images
+using Colors # for making images greyscale
+using Images # for channelview; converting images to matrices
+using ImageTransformations # for scaling high-quality images down
+
+#=
+Any one pixel in a given image has the value that is the sum of all of the pixels above it and to the left.
+Original    Integral
++--------   +------------
+| 1 2 3 .   | 1  3  6 .
+| 4 5 6 .   | 5 12 21 .
+| . . . .   | . . . . .
+=#
+
+function getImageMatrix(imageFile::AbstractString)
+    img = load(imageFile)
```

Comment:
I am not sure we need IO functionality in this library. There's nothing here specific to an integral array representation.